content/Hardware/Core Boards/How-to-burn-the-bootloader-on-an-Arduino-Mega-using-an-Arduino-UNO.md

```diff
@@ -44,11 +44,13 @@ The bootloader is a piece of code stored in a reserved space in your Arduino boa
 
 08. Connect the Arduino Uno to your PC again.
```

Comment:
```suggestion
08.  Connect the Arduino Uno to your PC again and make sure it's selected in `Tools > Port
```

Reply:
@seaxwi you are missing the closing backtick at the end of `` `Tools > Port` `` in your suggestion.

Reply:
@per1234 fixed